conda_build.recipe/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: numpy
+  version: 1.11.0{{ '+' + environ.get('GIT_DESCRIBE_NUMBER', '0') + '.' + environ.get('GIT_DESCRIBE_HASH', '0') if environ.get('GIT_DESCRIBE_NUMBER', '0') != '0' else "" }}
```

Comment:
The build you get out of the box from Anaconda is actually linked to
openblas; there's some extra step required to get MKL.
.
I'm general I agree that there's not much point in our trying to use conda
as a distribution method for official releases, because for anyone who's
bought into that ecosystem they might as well stick with the "official"
numpy builds. What would be valuable imho is if there's some way we can use
this to get more people to test pre releases ("Hi everyone, 1.11b1 is out!
Here's how to test it in one command: conda install ..."). I'm not sure how
this would actually work given conda's numpy abi enforcement though.
On Jan 17, 2016 07:21, "Ralf Gommers" <notifications@github.com> wrote:

> In conda_build.recipe/meta.yaml
> <https://github.com/numpy/numpy/pull/7036#discussion_r49947588>:
>
> > @@ -0,0 +1,29 @@
> > +package:
> > +  name: numpy
> > +  version: 1.11.0{{ '+' + environ.get('GIT_DESCRIBE_NUMBER', '0') + '.' + environ.get('GIT_DESCRIBE_HASH', '0') if environ.get('GIT_DESCRIBE_NUMBER', '0') != '0' else "" }}
>
> Anything that we would upload would by default not be visible to users as
> well as less optimized than the version that Anaconda itself ships (no
> MKL), right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numpy/numpy/pull/7036/files#r49947588>.
>